contracts/upgradeable_contracts/amb_erc20_to_native/HomeAMBErc20ToNative.sol

```diff
@@ -1,15 +1,17 @@
 pragma solidity 0.4.24;
 
 import "./BasicAMBErc20ToNative.sol";
+import "../../interfaces/IMediatorFeeManagerBothDirections.sol";
 import "../BlockRewardBridge.sol";
+import "../RewardableMediator.sol";
 
 /**
 * @title HomeAMBErc20ToNative
 * @dev Home mediator implementation for erc20-to-native bridge intended to work on top of AMB bridge.
 * It is design to be used as implementation contract of EternalStorageProxy contract.
 */
-contract HomeAMBErc20ToNative is BasicAMBErc20ToNative, BlockRewardBridge {
-    bytes32 internal constant TOTAL_BURNT_COINS = 0x17f187b2e5d1f8770602b32c1159b85c9600859277fae1eaa9982e9bcf63384c; // keccak256(abi.encodePacked("totalBurntCoins"))
+contract HomeAMBErc20ToNative is BasicAMBErc20ToNative, BlockRewardBridge, RewardableMediator {
+    bytes32 internal constant TOTAL_SUPPLY_COINS = 0xb32740a8fb91550d50618ece1336a41a472ba5977838b1cd512bba896c895fc6; // keccak256(abi.encodePacked("totalSupplyCoins"))
```

Comment:
Ok, I got it. I updated the logic, so that now fee manager is inherited by the primary mediator contract, so we can use `TOTAL_BURNT_COINS` as it was used before.